docs/user/analytics.rst

```diff
@@ -1,8 +1,7 @@
 How to use traffic analytics
 ============================
 
-In this guide, you can learn to use Read the Docs' built-in traffic analytics for your documentation project.
-You will also learn how to optionally add your own Google Analytics account or completely disable Google Analytics on your project.
+This guide covers traffic analytics for your documentation project, either built-in or provided through Read the Docs Addons.
```

Comment:
"Built-in" and "provided through Read the Docs Addons" is the same thing, actually. So, I would probably rewrite it as:

```suggestion
This guide covers traffic analytics for your documentation project provided through Read the Docs Addons.
```

Reply:
good bullshit detector there!